prisondersdilemma.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May  7 13:37:28 2020
+
+@author: katha
+"""
+
+from psychopy.visual import Window, TextStim, Circle, ImageStim, Rect, ShapeStim
+from psychopy.core import wait, Clock
+from psychopy.event import getKeys, clearEvents, waitKeys, Mouse
+from numpy.random import shuffle
+import numpy as np
+import random
+
+#these are all just building blocks for now, not functioning code! it won't run yet
+
+#create window
+
+DISPSIZE = (1000, 700)
+BGC = 'white'
+FGC = (-1,-1,-1) #defining colour of text shown => black
+
+win = Window(size=DISPSIZE, units='pix', color = BGC, fullscr=False)
+
+
+#create objects and stimuli 
+
+background_image = ImageStim(win, 'prisonwall.png', autoDraw=True) #maybe for works for using a background image?
+welcome_str = f'Imagine you are one of two people being questioned about the same crime.\n'\
```

Comment:
This is a great start and I really like your project overall! Perhaps you could start with an "Intro" page with a short title, your name, date and how long it takes to play the game. This would be a nice introduction before reading the instructions. See here for one example: https://ncase.me/trust/